composer.json

```diff
@@ -25,9 +25,10 @@
         "drupal/gin_toolbar": "^1.0@RC",
         "drupal/graphql": "^4.9",
         "drupal/graphql_compose": "^2.2",
-        "drupal/group": "^3.2",
+        "drupal/group": "^3.3",
         "drupal/login_gov": "^1.1",
         "drupal/next": "^2.0@beta",
+        "drupal/security_review": "^3.0",
```

Comment:
Are we doing something with this module? I don't see any config changes ( core.extension.yaml, module-specific configs, etc)

IF there ARE config changes and they got missed, we probably want to apply https://github.com/CivicActions/pgov-cms/pull/12 first , and rebase, so we're working with the same config dir.